src/step-14-typescript/config.ts

```diff
@@ -6,6 +6,7 @@ const schema = S.object()
   .prop('JWT_SECRET', S.string().required())
   .prop('LOG_LEVEL', S.string().default('info'))
   .prop('PRETTY_PRINT', S.string().default(true))
+  .valueOf()
```

Comment:
I wonder why this change is not needed in all the other config.js files

Reply:
Because it just have to do with typing. It was removed in fastify/env-schema#137. Another option would be to have this as any or to have another typescript workaround, but this seemed cleaner to me. I have opened an issue in fastify/env-schema#138 to get a proper solution